packages/frontend/package.json

```diff
@@ -4,8 +4,9 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
-    "start": "next start -p 80"
+    "build": "next build && cp -r public build/standalone/packages/frontend && cp -r build/static build/standalone/packages/frontend/build/static",
```

Comment:
I think we can omit this part here and move it to the copy instructions of the Dockerfile.
See line 57: https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile

Reply:
Or you could try to adapt most of the Dockerfile, as there are also some other performance gains.

This is what we use in another project

```Dockerfile
FROM <NODE_IMAGE>

RUN apk add --no-cache libc6-compat

WORKDIR /app

ENV NODE_ENV=production

RUN addgroup -g 1001 -S nodejs
RUN adduser -S nextjs -u 1001

# COPY --from=builder /app/public ./public     we dont have a public folder atm

# Automatically leverage output traces to reduce image size
# https://nextjs.org/docs/advanced-features/output-file-tracing
COPY --chown=nextjs:nodejs .next/standalone ./
COPY --chown=nextjs:nodejs .next/static ./.next/static

USER nextjs

ENV PORT 80

CMD node server.js
```

Reply:
I could but if you just like to use the build and startstandalone script without using the Dockerfile it wouldn't work anymore, but I guess it's a matter of preference.

Reply:
That is very true. A colleague of mine who is using a Windows computer could not set the environment variable PORT like you did in the script though. So we decided that we'd always execute the production build within a docker container locally.

Reply:
Makes sense! I changed it.